fhir-server-test/src/test/java/com/ibm/fhir/server/test/ConcurrentUpdateTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * (C) Copyright IBM Corp. 2017, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.server.test;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertNotNull;
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import javax.ws.rs.core.Response;
+
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.client.FHIRParameters;
+import com.ibm.fhir.client.FHIRResponse;
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.Patient;
+import com.ibm.fhir.model.test.TestUtil;
+
+/**
+ * Tests specifically the concurrent creation and updating of a single patient, with the updateCreate feature enabled.
+ * The idea is to ensure that no patients with duplicate logical ids are created.
+ *
+ * NOTE: This test is NOT included in the fhir-server testng suite because of an internal problem with Derby that
+ * surfaces after the first execution of this test against a clean Derby DB. Subsequent test runs of this class will
+ * fail with an XAException that has an unknown error code. It should only be run against a DB2 database.
```

Comment:
```suggestion
 *
 * NOTE: This test is NOT included in the fhir-server testng suite because of an internal problem with Derby that
 * surfaces after the first execution of this test against a clean Derby DB. Subsequent test runs of this class will
 * fail with an XAException that has an unknown error code. It should only be run against a DB2 database.
```

Do we need this comment?

Reply:
yes, good call

Reply:
I updated the javadoc accordingly